lib/Service/Sync/ImapToDbSynchronizer.php

```diff
@@ -131,6 +136,8 @@ public function syncAccount(Account $account,
 				$rebuildThreads = true;
 			}
 		}
+		// disconnect imap client
```

Comment:
```suggestion
```
https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/#h2-b7593c698a940